packages/side-nav/src/vaadin-side-nav-item.js

```diff
@@ -189,13 +192,21 @@ class SideNavItem extends SideNavChildrenMixin(ElementMixin(ThemableMixin(Polyli
   render() {
     return html`
       <div part="content" @click="${this._onContentClick}">
-        <a id="link" href="${ifDefined(this.path)}" part="link" aria-current="${this.current ? 'page' : 'false'}">
+        <a
+          id="link"
+          ?disabled="${this.disabled}"
+          tabindex="${this.disabled ? '-1' : '0'}"
+          href="${ifDefined(this.path)}"
```

Comment:
Make sure to also remove the href once the link is disabled. See for example this flow commit and corresponding issue: https://github.com/vaadin/flow/commit/fb35aeb98a50105856be84a4383769f8a8c33102

Reply:
Thanks, done.